### ISSUE TITLE
unable to import packages using git+ssh

### ISSUE THREAD

**Issue by ivan1986** [yarnpkg/yarn#3580]
**Do you want to request a *feature* or report a *bug*?**

Bug

**What is the current behavior?**

```
error Command failed.
Exit code: 128
Command: git
Arguments: clone ssh://git@github.com/webpack/webpack.git#3d653290fafe385277b48e5a36807124618b9561 /home/ivan/.cache/yarn/v1/.tmp/52478823f6da55becd4cd5a31f77c742
Directory: /home/ivan/projects/x
Output:
Клонирование в «/home/ivan/.cache/yarn/v1/.tmp/52478823f6da55becd4cd5a31f77c742»…
fatal: remote error: 
   is not a valid repository name
  Email support@github.com for help
info Visit https://yarnpkg.com/en/docs/cli/import for documentation about this command.
```

**If the current behavior is a bug, please provide the steps to reproduce.**
use example package.json
```
{
          "name": "test",
          "version": "1.0.0",
          "description": "test",
          "dependencies": {
                  "webpack": "git+ssh://git@github.com:webpack/webpack.git"
          }
}
```
clear yarn cache, run npm install, run yarn import

**What is the expected behavior?**
success Saved lockfile.

**Please mention your node.js, yarn and operating system version.**
Node v6.10.3, Yarn 0.24.5, linux 4.10.0

**Comment by thevtm** [comment]
It only does that with password protected keys.
If you remove the password It should work.
`openssl rsa -in ~/.ssh/id_rsa -out ~/.ssh/id_rsa_new`

**Comment by BYK** [comment]
Closing due to inactivity. Also, this should be solved post 1.0. Please file a new issue if not.
